tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PitRunConfiguration.java

```diff
@@ -16,32 +16,37 @@
 
 package org.pitest.pitclipse.ui.behaviours.pageobjects;
 
-import com.google.common.collect.ImmutableList;
+import static com.google.common.collect.ImmutableList.copyOf;
+import static org.pitest.pitclipse.runner.config.PitConfiguration.DEFAULT_AVOID_CALLS_TO_LIST;
 
 import java.util.List;
 
-import static com.google.common.collect.ImmutableList.copyOf;
-import static org.pitest.pitclipse.runner.config.PitConfiguration.DEFAULT_AVOID_CALLS_TO_LIST;
+import com.google.common.collect.ImmutableList;
 
 public class PitRunConfiguration {
 
     private final String name;
     private final List<String> projects;
+    private final String testObject;
     private final boolean runInParallel;
     private final boolean incrementalAnalysis;
     private final String excludedClasses;
     private final String excludedMethods;
     private final String avoidCallsTo;
+    private final boolean testClass;
```

Comment:
can you please put some comments about these two new fields?

Reply:
Yes